debian/daos-server-tests.install

```diff
@@ -8,3 +8,7 @@ usr/bin/vea_ut
 usr/bin/srv_checksum_tests
 usr/bin/vos_tests
 usr/bin/vea_stress
+usr/bin/vos_perf
+usr/bin/daos_run_io_conf
+usr/bin/obj_ctl
+usr/bin/daos_gen_io_conf
```

Comment:
Do these need `libdpar.so` also?  I don't see how they are going to get it if so.  I don't see any dependency on `daos-client-tests` nor am I convinced there should be one.

Reply:
They would need libdpar.so (or libdpar_mpi.so) to run, yes.   Maybe there is a more appropriate place to put that library.

I'm not sure I got the packaging right here which is why I added you to the patch.

Reply:
> They would need libdpar.so (or libdpar_mpi.so) to run, yes

Does that mean every consumer needs to specify an `LD_LIBRARY_PRELOAD` whether they want `libdpar.so` or `libdpar_mpi.so` or is the former a default without an `LD_LIBRARY_PRELOAD`?

> Maybe there is a more appropriate place to put that library.

There should be a "common" (i.e. `daos-tests-common`, probably) file that both packages depend on that would have the common _things_ (these libraries at least) in them

Reply:
By default, everything is linked directly with libdpar.so.  If they want to use the MPI version, they do

mpirun -x LD_PRELOAD=libdpar_mpi.so [args] ./daos_test -i

The mpi version is just a drop-in replacement for the same interfaces.

Reply:
@brianjmurrell I may need your help with the spec changes if we do more there